### PR TITLE
Verify AdoptOpenJDK.md tag and handle <build>_adopt tags

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: andrew-m-leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

**Comment by karianna** [comment]
Is this really WIP?

**Comment by andrew-m-leonard** [comment]
hi Martijn, yes, but only as I haven't had a chance to test it fully yet.. back in work monday so will do it then, cheers

File: sbin/prepareWorkspace.sh [hunk 1]
```diff
@@ -100,6 +100,14 @@ checkoutAndCloneOpenJDKGitRepo()
     git reset --hard "origin/${BUILD_CONFIG[BRANCH]}"
   fi
 
+  if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ] && ( [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK11_CORE_VERSION}" ] || [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK12_CORE_VERSION}" ] || [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK13_CORE_VERSION}" ] || [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDKHEAD_CORE_VERSION}" ]) ; then
```
Comment (johnoliver) [inline sbin/prepareWorkspace.sh]: can you change this to:
```
if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ] && [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 11 ]
```

Comment (andrew-m-leonard) [reply sbin/prepareWorkspace.sh]: fixed

**Pull request merged**